src/PriceOracle.sol

```diff
@@ -111,22 +121,42 @@ contract PriceOracle is AccessControl {
                 usedRedundancy = maxConsideredRedundancy;
             }
 
-            // use the increaseRate array of constants to determine
+            // Set the number of rounds that were skipped
+            uint256 skippedRounds = currentRoundNumber - lastClaimedRound - 1;
+
+            // Use the increaseRate array of constants to determine
             // the rate at which the price will modulate - if usedRedundancy
             // is the target value 4 there is no change, > 4 causes an increase
             // and < 4 a decrease.
-            uint256 ir = increaseRate[usedRedundancy];
-
             // the multiplier is used to ensure whole number
+
+            // We first apply the increase/decrease rate for the current round
+            uint256 ir = increaseRate[usedRedundancy];
             currentPrice = (ir * currentPrice) / multiplier;
 
-            //enforce minimum price
+            // If previous rounds were skipped, use MAX price increase for the previouse rounds
+            if (skippedRounds > 0) {
```

Comment:
This would seem to protect against round number going backwards, but I think it would be best to reject it with a > check above.

Reply:
We could use > as a double insurance, yes.